docker_build.mk

```diff
@@ -15,7 +15,6 @@ uncached-docker-build-toolchain:
 docker-build-all:
 	docker run --rm -v "$(CURDIR)/builds:/builds" \
 					-v "$(CURDIR):/solo-base:ro" \
-					-u $(shell id -u ${USER}):$(shell id -g ${USER}) \
```

Comment:
This one is handy for a local run so the resulted files were written with running user privileges, although it was not working lately. Can you fix that? If not, please leave it as a commented line.

Reply:
done